sites/search/json/canada-en.json

```diff
@@ -0,0 +1,487 @@
+{
+  "@context": {
+    "dc": "http://purl.org/dc/terms/",
+    "suggestions": {
+      "@id": "http://www.w3.org/2001/XMLSchema#string",
+      "@container": "@list"
+    }
+  },
+  "@language": "en",
+  "dc:title": "Global top search term",
+  "dc:modified": "2018-02-06",
```

Comment:
Is that the real date we have reviewed the top search term list? If so, it is outdated for sure.

Reply:
I downloaded the files from AEM. I am not sure if our team is responsible for updating the content of the document.

Reply:
I have asked the analytic team to provide a list of most popular search terms.

Reply:
You will need to find out the process of who and how that file is updated. And then ensure the person that are making the update are also updating the modified date. And let that person know to update/review the title too.
Is it updated by analytic team??? They should not update/manage any content on AEM, usually that is done by our publisher or in rare situation by the Dev team